Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -418,9 +418,12 @@ AccessibilityRole AccessibilityNodeObject::determineAccessibilityRoleFromNode(Tr
         return AccessibilityRole::Footer;
     }
 
-    // menu tags with toolbar type should have Toolbar role.
-    if (node()->hasTagName(menuTag) && equalLettersIgnoringASCIICase(getAttribute(typeAttr), "toolbar"_s))
-        return AccessibilityRole::Toolbar;
+    // menu tags with toolbar type should have Toolbar role and menu tags without toolbar should have role list
```

Comment:
I guess explain that the HTML AAM spec says to always map to role list and that special handling for type="toolbar" is only for backward compatibility.